gooddata-java-model/src/main/java/com/gooddata/sdk/model/md/maintenance/ExportProjectArtifact.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2004-2020, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.model.md.maintenance;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.gooddata.sdk.model.gdc.UriResponse;
+
+/**
+ * Complete project export result structure.
+ * For internal use only.
+ * Deserialization only.
+ */
+@JsonTypeName("exportArtifact")
+public class ExportProjectArtifact extends PartialMdArtifact {
```

Comment:
Hmm, we should reconsider if these internal classes are part of the model or not.

Reply:
Well, this one is not really internal, it's part of the API, so I would say that it should be in the model (one may want to reuse it for API implementation).